### DISCUSSION TITLE
Tidak bisa terhubung dengan Perangkat (Please Scan Code Again)

### DISCUSSION THREAD

**Discussion by ambimuhammad** [kirimwa-id/forum#114] [Tanya Jawab] [answered]
Salam Mas Rio,

Saya ada kendala ketika menghubungkan dengan perangkat
saya sudah dapet qr codenya tapi ketika discan selalu logging in, dan kemudian please code again

proses scan sudah dibawah 10 detik

seperti gambar berikut
![Screenshot_2022-04-22-10-26-37-991_com whatsapp](https://user-images.githubusercontent.com/11288607/164590752-bf359b72-594d-469b-8618-baf08a530610.jpg)

**Comment by melodiazz** [comment, marked as answer]
Kemungkinan besar sudah menggunakan versi whatsapp terbaru yang hanya bisa menggunakan Multi Device.

Fitur untuk support multi device masih on progress as in https://github.com/kirimwa-id/forum/discussions/42

**Reply by ambimuhammad** [reply]
oh seperti itu Mas, terima kasih infonya Mas :)
